build.gradle

```diff
@@ -172,8 +176,7 @@ ext {
     ]
 }
 
-apply from: "gradle/buildScanUserData.gradle"
-apply from: "gradle/taskCacheBuildScanUserData.gradle"
+apply plugin: "buildscan-configuration"
```

Comment:
Why don't we apply that plugin via the `plugins` block?

Reply:
In general we don't want to rework how the root build.gradle script is behaving right now and therefore want to preserve the order in which everything is configured. Once we have refactored the logic in gradle/gradle into plugins, the root build.gradle is next.